### PR TITLE
[DOC] Use relative paths instead of absolute paths

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

1. Use relative paths instead of absolute paths, avoid jumping to the latest document page.
2. fix `setting` to `settings`

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [X] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/quick_start/quick_start_with_datagrip.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 ### Get DataGrip And Install
 Please go to [Download DataGrip](https://www.jetbrains.com/datagrip/download) to get and install an appropriate version for yourself.
 ### Get Kyuubi Started
-[Get kyuubi server started](https://kyuubi.apache.org/docs/latest/quick_start/quick_start.html) before you try DataGrip with kyuubi.
+[Get kyuubi server started](../quick_start/quick_start.md) before you try DataGrip with kyuubi.
```
Comment (ulysses-you) [inline docs/quick_start/quick_start_with_datagrip.md:26]: `./quick_start.html` ?

Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:26]: We currently do not have a uniform suffix, using `. Md` works without `#` behind.

Comment (ulysses-you) [reply docs/quick_start/quick_start_with_datagrip.md:26]: oh sorry, I mean `./quick_start.md`

Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:26]: Now Kyuubi Doc is having a problem with 404 when it needs to specify a title to jump to if the suffix is `.md`.
So, do we need to unify the suffix, and if we do, it's just `.html`.

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**